tests/test_simulator.py

```diff
@@ -7,8 +7,11 @@
 from spectractor.simulation.image_simulation import ImageSim
 from spectractor.config import load_config
 import os
+import unittest
 
 
+# TODO: DM-33441 Fix broken spectractor tests
+@unittest.skipIf(True, 'Skipping to avoid libradtran dependency')
```

Comment:
Ordinarily you would try to import the code that uses libradtran and on ImportError set a flag so that this could be skipped based on knowledge of the environment. Why can't you do that here?

Reply:
It's a few lines in before it fails, and I can't see an easy way of of working that out without actually running some of this test code. There are env vars you can test which indicate whether it _should_ be available, but installs can easily be broken, and this isn't a true dependency, in that we don't use the simulation tools in the AuxTel pipeline, so I think just blanket-skipping these is fine for now.

Reply:
I think @timj doesn't realize just how hinky the connection to libradtran is.  It's not a Python package to import; it's a binary (`uvspec`) that is executed via `subprocess` and assumed to be in the `HOME` directory...